addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -401,10 +402,13 @@ def _import_fattura_pa(self, tree, invoice):
             company = elements and self.env['res.company'].search([('vat', 'ilike', elements[0].text)], limit=1)
             if not company:
                 elements = tree.xpath('//CessionarioCommittente//CodiceFiscale')
-                company = elements and self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
+                if not elements:
+                    message_to_log.append(_('No Codice Fiscale found in the xml.'))
+                    continue
+                company = self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
                 if not company:
                     # Only invoices with a correct VAT or Codice Fiscale can be imported
-                    _logger.warning('No company found with VAT or Codice Fiscale like %r.', elements[0].text)
+                    message_to_log.append(_('No company found with VAT or Codice Fiscale like %r.', elements[0].text))
```

Comment:
@JulienVR Not sure where it would be logged because it could not be posted on any object.